packages/loopring_v3/contracts/amm/libamm/AmmExitProcess.sol

```diff
@@ -51,6 +51,9 @@ library AmmExitProcess
         if (signature.length == 0) {
             if (S.exitMode) {
                 require(exit.fee == 0, "INVALID_FEE");
+                // for (uint i = 0; i < S.tokens.length; i++) {
```

Comment:
Do we need this?

Reply:
Shouldn't be needed because `tokenBalancesL2` contains the `balancesL1` values to calculate the exit amounts so if there are enough funds in the pool on L2 it should work correctly.

Reply:
OK